tests/puppeteer/pages/BO/advancedParameters/team/addEmployee.js

```diff
@@ -0,0 +1,56 @@
+require('module-alias/register');
+const BOBasePage = require('@pages/BO/BObasePage');
+
+module.exports = class AddPageEmployee extends BOBasePage {
+  constructor(page) {
+    super(page);
+
+    this.pageTitleCreate = 'Add new • prestashop';
```

Comment:
```suggestion
    this.pageTitleCreate = 'Add new •';
```
prestashop depends on shopname

Reply:
did you apply this modification, can I resolve conversation